charts/cspm-collector/templates/clusterrole.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if and .Values.rbac.create .Values.deploy  }}
```

Comment:
the `deploy` flag is probably unnecessary, since a user could just as easily not deploy the chart 😄 

this can be handled by a condition in the parent chart, as we do for the current subcharts

Reply:
removed :)